src/app/interceptors/auth.interceptor.ts

```diff
@@ -0,0 +1,30 @@
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
+import { Observable } from 'rxjs';
+import { ApiService } from '../services/api/api.service';
+
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
+    constructor(private localStorage: LocalStorageService, private sessionStorage: SessionStorageService) {}
+
+    intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+        if (
+            ! request
+            || !request.url
```

Comment:
I'd put the first two conditions on the same line. Otherwise, it's just wasted space.

Reply:
Fixed with linter